wolfssl/ssl.h

```diff
@@ -4202,13 +4203,18 @@ WOLFSSL_API void wolfSSL_CTX_SetPerformTlsRecordProcessingCb(WOLFSSL_CTX* ctx,
 
     WOLFSSL_API int wolfSSL_CertManagerLoadCA(WOLFSSL_CERT_MANAGER* cm,
         const char* f, const char* d);
+    WOLFSSL_API int wolfSSL_CertManagerLoadCABuffer_ex2(WOLFSSL_CERT_MANAGER* cm,
```

Comment:
This would be the first _ex2 in wolfssl/ssl.h that I can see. Could we name it something like wolfSSL_CertMangerLoadCaBufferType ? That's personal preference though _ex2 also works.